docs/content/connectors/oracle-cdc.md

```diff
@@ -231,6 +231,45 @@ Connector Options
 </table>    
 </div>
 
+Available Metadata
+----------------
+
+The following format metadata can be exposed as read-only (VIRTUAL) columns in a table definition.
+
+<div class="highlight">
```

Comment:
We don't need the outer `div`, otherwise, there is a border around the table. This problem also exists in other connectors.

![image](https://user-images.githubusercontent.com/5378924/140714060-24c23f59-73ab-4870-9c59-00f690fe1cca.png)